developer/configuration.md

```diff
@@ -25,6 +25,13 @@ For convenience, the initial Reaction administrator can be configured here.
 
 Creating a `settings.json` will prevent the default `dev.settings.json` from being loaded when you use the `reaction` command to start Reaction.
 
+Once you have edited the **_settings/settings.json_** you will need to run:
+```sh
+reaction reset && rc 
```

Comment:
my apologies to you both ;-) . `rc` is a bit of an undocumented shortcut for `reaction`.  They are interchangeable on the command line.   We _should_ mention that somewhere, and in the docs. however, in the example here we can just say `reaction`.  I guess the main thing was to mention the you need to reset, if you want to recreate the initial default user.